core/mondoo-tls-security.mql.yaml

```
@@ -93,70 +110,166 @@ queries:
         default: score(0);
       }
   - uid: mondoo-tls-security-cert-no-certs-expired
-    title: None of the certificates (intermediate, root) have expired
+    title: Ensure none of the certificates (intermediate, root) have expired
```

```suggestion
    title: None of the certificates (intermediate or root) have expired
```
No other titles start with "Ensure" -- which is a good thing! Eventually no policies will have that.